pkg/rancher-desktop/backend/lima.ts

```diff
@@ -1558,7 +1558,7 @@ export default class LimaBackend extends events.EventEmitter implements VMBacken
     if (bridgedIP) {
       console.log(`Using ${ bridgedIP } on bridged network rd0`);
 
-      return 'rd0';
+      return { iface: 'rd0', addr: bridgedIP };
```

Comment:
nit; I feel like the function comments might need to be updated since it no longer just gets the `Get the network interface to listen on for services....`, it also returns the addr too. Also, some additional comments on the actual function would be great.